lib/mono/changeset.rb

```diff
@@ -135,21 +135,29 @@ def bump_index
     end
 
     def date
-      commit[:date]
+      commits.first[:date]
     end
 
-    def commit
-      @commit ||=
+    def commits
+      @commits ||=
         begin
           escaped_path = path.gsub('"', '\"')
-          git_log =
-            `git log -n 1 --pretty="format:%h %H %cI" -- "#{escaped_path}"`
-          short, long, date = git_log.split(" ")
-          {
-            :short => short,
-            :long => long,
-            :date => Time.parse(date)
-          }
+          cmd = <<~COMMAND
+            git log \
+              --pretty="format:%h %H %cI" \
+            --grep="\\[skip mono\\]" \
```

Comment:
```suggestion
            --grep="\\[skip mono\\]" -i \
```

Makes the grep case-insensitive (I think we already do ignore case for other `[skip ...]` things? If not feel free to ignore this)

Reply:
We don't case insensitve check anything else that I'm aware. So I'm okay with leaving it like this.